lib/remote/jsonrpcconnection.cpp

```diff
@@ -208,18 +208,8 @@ void JsonRpcConnection::Disconnect()
 		JsonRpcConnection::Ptr keepAlive (this);
```

Comment:
... ASAP?

Reply:
I don't think that would be useful. We're only concerned with whether the I/O threads are executing the coroutines correctly.

Reply:
And that's why I think we should, while on it, log this even before the coroutine spawns.